### PR TITLE
Pull out resource meta for duty calculator

### PR DESCRIPTION
**Pull request by willfish**
### Jira link

HOTT-<TODO>

### What?

I have added/removed/altered:

- [x] Adds support for deconstructing meta so that downstream models can have `Commodity#meta` called against them 

### Why?

I am doing this because:

- This is needed to make decisions about questions the duty calc asks

### CODE REVIEW

File: lib/uktt/parser/json_api.rb [hunk 1]
```diff
@@ -29,8 +29,8 @@ def parse_resource(resource)
         result = {}
 
         parse_top_level_attributes!(resource, result)
-
         parse_relationships!(resource['relationships'], result) if resource.key?('relationships')
+        parse_meta!(resource, result)
```
Comment (willfish) [inline lib/uktt/parser/json_api.rb:33]: Following the same primadonna idiom for now. I think we'll probably refactor this class at some stage.

Comment (tavyy) [reply lib/uktt/parser/json_api.rb:33]: Do we have a unit test that could indirectly test this?

**Comment by willfish** [comment]
```
Finished in 1.27 seconds (files took 0.50266 seconds to load)
49 examples, 0 failures
```

**Comment by tavyy** [comment]
We don't use yet CircleCi here but that'll be fixed. I have seen the tests green.

**Review by tavyy — APPROVED** [review]

**Pull request merged**